modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/partner-portal/fragments/mdf-claim-details-summary/index.js

```diff
@@ -56,4 +58,4 @@ const formatCurrency = (value) =>
 		style: 'currency',
 	}).format(value);
 
-updateMDFClaimSummary();
+getMDFClaimSummary();
```

Comment:
if (layoutMode !== 'edit') {
     getMDFClaimSummary();
}

add this verification to follow the pattern